src/routes/articles.route.js

```diff
@@ -49,6 +56,18 @@ const articleRouter = Router();
  *       500:
  *         description: ran
  */
-articleRouter.post('/', create);
+articleRouter.post(
+  '/',
+  validatorInput(articleSchema),
+  isAuthorized,
+  createArticle,
+);
+
+articleRouter.put(
```

Comment:
please add doc for this route.

Reply:
I have added doc. for this route.
thanks for catching that.